Readme.md

```diff
@@ -52,17 +65,17 @@ class BundleInitializationTest extends BaseBundleTestCase
 
     public function testBundleWithDifferentConfiguration()
     {
-        // Create a new Kernel
-        $kernel = $this->createKernel();
-
-        // Add some configuration
-        $kernel->addConfigFile(__DIR__.'/config.yml');
-
-        // Add some other bundles we depend on
-        $kernel->addBundle(OtherBundle::class);
-
         // Boot the kernel as normal ...
-        $this->bootKernel();
+        $kernel = self::bootKernel([
+            'bundles' => [
+                // Add some other bundles we depend on
+                OtherBundle::class,
+            ],
+            'configFiles' => [
+                // Add some configuration
+                __DIR__.'/config.yml',
```

Comment:
Can we keep `$kernel->addConfigFile(__DIR__.'/config.yml');`?

Reply:
Sure. :)